xinference/model/llm/llm_family.json

```diff
@@ -3930,5 +4026,281 @@
         "<start_of_turn>"
       ]
     }
+  },
+  {
+    "version": 1,
+    "context_length": 4096,
+    "model_name": "Platypus2-70B-instruct",
+    "model_lang": [
+      "en"
+    ],
+    "model_ability": [
+      "generate"
+    ],
+    "model_description": "Platypus-70B-instruct is a merge of garage-bAInd/Platypus2-70B and upstage/Llama-2-70b-instruct-v2.",
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 70,
+        "quantizations": [
+          "none"
+        ],
+        "model_id": "garage-bAInd/Platypus2-70B-instruct",
+        "model_revision": "31389b50953688e4e542be53e6d2ab04d5c34e87"
+      }
+    ]
+  },
+  {
+    "version": 1,
+    "context_length":2048 ,
+    "model_name": "Aquila2",
+    "model_lang": [
+      "en",
+      "zh"
+    ],
+    "model_ability": [
+      "generate"
+    ],
+    "model_description": "Aquila2 series models are the base language models",
+    "model_specs": [
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 7,
+        "quantizations": [
+          "none"
+        ],
+        "model_id": "BAAI/Aquila2-7B",
+        "model_revision": "9c76e143c6e9621689ca76e078c465b0dee75eb8"
+      },
+      {
+        "model_format": "pytorch",
+        "model_size_in_billions": 34,
+        "quantizations": [
+          "none"
+        ],
+        "model_id": "BAAI/Aquila2-34B",
+        "model_revision": "356733caf6221e9dd898cde8ff189a98175526ec"
+      }
+    ]
+  },
+  {
+    "version": 1,
+    "context_length": 2048,
+    "model_name": "AquilaChat2",
```

Comment:
This can have gguf / gptq/ awq model. See https://huggingface.co/TheBloke?search_models=aqu